server/src/test/java/io/crate/expression/scalar/string/TrimFunctionTest.java

```diff
@@ -148,10 +148,21 @@ public void testEvaluateTrimWithoutCharsToTrim() {
     @Test
     public void testEvaluateNullInputOptimisedTrim() {
         assertEvaluateNull("trim(name)", Literal.of(DataTypes.STRING, null));
+        assertEvaluateNull("trim('foo' FROM name)", Literal.of(DataTypes.STRING, null));
         assertEvaluateNull("ltrim(name)", Literal.of(DataTypes.STRING, null));
         assertEvaluateNull("rtrim(name)", Literal.of(DataTypes.STRING, null));
+        assertEvaluateNull("btrim(name)", Literal.of(DataTypes.STRING, null));
         assertEvaluateNull("ltrim(name, null)", Literal.of(DataTypes.STRING, null));
         assertEvaluateNull("rtrim(name, null)", Literal.of(DataTypes.STRING, null));
+        assertEvaluateNull("btrim(name, null)", Literal.of(DataTypes.STRING, null));
```

Comment:
Are these 3 new asserts just to cover all variations? They seem unrelated to the bug fix.

Reply:
yep!

Reply:
Please take on more look (squashed commit already)
I change the 2nd arg on this method from `null` to `foo`, to separate visibly that the `null` on the 2nd arg is tested in the new method. thx!

Reply:
Looks good 👍